applications/ironside/se/update/README.rst

```diff
@@ -0,0 +1,77 @@
+.. _ironside_se_update_application:
+
+IRONside SE Update
+##################
+
+The *IRONside SE Update* application updates the IRONside SE firmware on a SoC that already has IRONside SE installed.
+It can update both the main image and the recovery image of IRONside SE using the IRONside SE firmware release ZIP file.
+
+Update procedure
+****************
+
+The application invokes the IRONside SE update service and passes the parameters that correspond to the location of the HEX file of the IRONside SE firmware update.
+
+The application prints the return value of the service call and outputs information from the update HEX file.
+
+After the service call completes, the IRONside SE firmware updates the internal state of the device.
+The firmware installs the update during the next device boot.
+This operation can take several seconds.
+
+Once the operation has completed the boot report can be read to verify that the update has taken place.
+
+Requirements
+************
+
+The firmware supports the following development kit:
+
+.. table-from-sample-yaml::
+
+
+Building and running
+********************
+
+This application can only be used when there is already a version of IRONside SE installed on the device.
+
+A version of this application is bundled with the IRONside SE release.
```

Comment:
So which versions should they use? The one bundled with the ironside release or this one? @hakonfam

Reply:
They can choose, but I think its valuable to describe this here in case they stumble upon this sample in an attempt to perform the update.

Reply:
I removed the description of the script.